core/chainio/avs_writer.go

```diff
@@ -2,15 +2,25 @@ package chainio
 
 import (
 	"context"
+	"fmt"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/avsregistry"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
 	"github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/Layr-Labs/eigensdk-go/signer"
+	gethcommon "github.com/ethereum/go-ethereum/common"
 	gethtypes "github.com/ethereum/go-ethereum/core/types"
 	servicemanager "github.com/yetanotherco/aligned_layer/contracts/bindings/AlignedLayerServiceManager"
 	"github.com/yetanotherco/aligned_layer/core/config"
 	"github.com/yetanotherco/aligned_layer/core/utils"
+	"math/big"
+	"time"
+)
+
+const (
+	LowFeeMaxRetries          = 25
+	LowFeeSleepTime           = 5 * time.Second
```

Comment:
```suggestion
	LowFeeMaxRetries          = 25
	LowFeeSleepTime           = 35 * time.Second
```

Reply:
Solved in [8749eb4](https://github.com/yetanotherco/aligned_layer/pull/330/commits/8749eb49181b7a0b40149123a886b0545ddd750f).